### ISSUE TITLE
Addition of stack_merge function.

### ISSUE THREAD

**Issue by SharonIV0x86** [KaisenAmin/c_std#22]
Been using c_std library for sometime I would like to propose the addition of a ``stack_merge`` function to the library, which would allow users to merge two stacks. This function should provide the option to either return a new stack or perform an in-place merge of stack2 into stack1. 

@KaisenAmin 
 
If this issue is assigned to me i will make sure to 
- Implement the said functionality and feature
- Follow contributing guidelines and coding standards. 
- Provide thorough documentation and test cases to demonstrate the functionality and reliability of the new feature.

**Comment by maxcomperatore** [comment]
The owner disappeared, I think we might fork this and continue the development.

**Comment by SharonIV0x86** [comment]
> The owner disappeared, I think we might fork this and continue the development.

I kinda wanted to contribute to this library as my major contribution to this. I have connection with the owner on Linkedin should i reach out to him there?

**Comment by maxcomperatore** [comment]
Yeah sure, I think he abandoned the project. Just hmu when you fork or when you want me to do it and we continue this.

**Comment by SharonIV0x86** [comment]
> I think he abandoned the project


Last i remember he was pretty active regarding his project on Linkedin and has made several posts of its existing and new features. Anyways i will fork it and work on it sideways.
